libs/core/src/scss/base/_variables.scss

```diff
@@ -87,19 +87,13 @@ $border-radius-round: 999px !default;
 /* Breakpoints
 ****************************************************************************/
 $breakpoints: (
-  small: 321px,
-  medium: 721px,
-  large: 1025px,
-  xlarge: 60em,
-  xxlarge: 80em,
+  xsmall: 0,
+  small: 632px,
```

Comment:
small breakpoint needs to be preserved, as we still want to support column widths (and screens) as low as 320px. width. We currently use this breakpoint internally (see image) and we (aligned with @henrikvoetmand) propose to move all breakpoints on step and reintroduce small: 321px.
<img width="504" alt="image" src="https://user-images.githubusercontent.com/42470636/235155092-9a4bdd92-46df-4784-ae2f-7b4c9dc3d96e.png">

Reply:
If we do this, we need to adjust the mapping in _grid.scss so it stays the same. It will look a bit weird, but I think it is the best solution to not break anything:

```
  'xs': 0,
  'sm': medium,
  'md': large,
  'lg': xlarge,
  'xl': xxlarge,

```

Reply:
I have changed the map in _grid.scss as you suggested here, but I didn't include `'xs': 0`. Setting the xs size to 0 in this map doesn't work, because it is not a value in the `$breakpoints` map in `_variables.scss`.

I have created this function instead:

```
@function get-breakpoint($breakpoint) {
  @if $breakpoint == 'xs' {
    @return 0;
  }

  $breakpoint-map-size: map.get($grid-sizes-to-breakpoint-map, $breakpoint);

  @return map.get(utils.$breakpoints, $breakpoint-map-size);
}
```

And use it here:

```
@mixin breakpoint-style($breakpoint) {
  @media (min-width: get-breakpoint($breakpoint)) { <--------------- HERE
    @include column-style($breakpoint, 1);
    @include column-style($breakpoint, 2);
    @include column-style($breakpoint, 3);
    @include column-style($breakpoint, 4);
    @include column-style($breakpoint, 5);
    @include column-style($breakpoint, 6);
    @include column-style($breakpoint, 7);
    @include column-style($breakpoint, 8);
    @include column-style($breakpoint, 9);
    @include column-style($breakpoint, 10);
    @include column-style($breakpoint, 11);
    @include column-style($breakpoint, 12);
  }
}
```